### PR TITLE
add ci operator file of v2023.1

### PR DESCRIPTION
**Pull request by rpancham**
Add openvino model server ci operator config file of v2023.1

### CODE REVIEW

File: ci-operator/config/opendatahub-io/openvino_model_server/opendatahub-io-openvino_model_server-v2023.1-release.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+binary_build_commands: 'mkdir -p .workspace; echo {\"gitref\": \"$(git rev-parse \
+  --short HEAD)\"} > .workspace/metadata.json'
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+  use_build_cache: true
+images:
```
Comment (Xaenalt) [inline ci-operator/config/opendatahub-io/openvino_model_server/opendatahub-io-openvino_model_server-v2023.1-release.yaml:9]: Need to add:

      buildArgs:
        - name: NVIDIA
          value: '1'
        - name: BASE_IMAGE
          value: 'nvcr.io/nvidia/cuda:11.8.0-runtime-ubi8'
        - name: RELEASE_BASE_IMAGE
          value: 'nvcr.io/nvidia/cuda:11.8.0-runtime-ubi8'
        - name: BASE_OS
          value: redhat
        - name: PROJECT_VERSION
          value: 2023.1.0
        - name: ov_source_branch
          value: releases/2023/1
        - name: ov_source_org
          value: openvinotoolkit
        - name: ov_contrib_branch
          value: releases/2023/1
        - name: ov_contrib_org
          value: openvinotoolkit
        - name: ov_use_binary
          value: '0'
        - name: sentencepiece
          value: '1'
        - name: CHECK_COVERAGE
          value: '1'
        - name: RUN_TESTS
          value: '1'
        - name: debug_bazel_flags
          value: '--strip=always'
        - name: CMAKE_BUILD_TYPE
          value: Release
        - name: GPU
          value: '1'

**Comment by Xaenalt** [comment]
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-v2023.1-release-images

**Comment by rpancham** [comment]
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-v2023.1-release-images

**Comment by Xaenalt** [comment]
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-v2023.1-release-images

**Comment by danilo-gemoli** [comment]
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-v2023.1-release-images

**Comment by Xaenalt** [comment]
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-v2023.1-release-images

**Comment by rpancham** [comment]
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-2023.1-release-images

**Comment by rpancham** [comment]
/pj-rehearse refresh

/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-2023.1-release-images

**Comment by Xaenalt** [comment]
/pj-rehearse pull-ci-opendatahub-io-openvino_model_server-2023.1-release-images

**Comment by Xaenalt** [comment]
WOOO build succeeded LETSGO :D

**Pull request merged**